index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Lobster&display=swap" rel="stylesheet">
+</head>
+<section>
+<body>
+    <div class="nav">
+   <ul>
+    <li><label class="logo">MY LOGO</label></li>
+    <li class="menubar"><i class="fa fa-bars"></i></li>
+   </ul>
+</div> 
+<div class="flexcontainer">
+    <div class="flexitem heading"><h1>Hey there. I’m Afreen.<br><b>I’m a software developer.</b></h1></div>
+    <div class="flexitem paragraph"><p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+        don’t hesitate to contact me.</p></div>
+    <div class="flexitem socialmediaicons">
+     <a href="https://www.twitter.com/tajulafreen"><i class="fa fa-twitter"></i></a>
+     <a href="https://www.facebook.com"><i class="fa fa-facebook"></i></a>
+     <a href="https://www.linkedin.com/in/tajul-afreen-shaik-843951251/"><i class="fa fa-linkedin"></i></a>
+     <a href="https://www.github.com/tajulafreen"><i class="fa fa-github"></i></a>
+     <a href="https://www.google.com"><i class="fa fa-google"></i></a>
+    </div>
+</div>
```

Comment:
Please I'd suggest you make the following changes 😃  ;

- [ ] The headline section should be wrapped in a `section` container as it represents a good HTML semantics for your project 

- [ ] It is really important to add the required background image for your project as there is none showing unlike what 
   we have in the design,you should use a `background :url()` property to import the images in your css styling 👍

- [ ]  You did a good job showing the social icon using an `a` tag with the `href` attribute, however  this should be enclosed in a `ul` tag with an `li` for each of them .
like this 👍 
    

>      <ul>
              <li><a href= ""></a></li>
                         .
                         .
                        . 
         </ul>